api/views/schema.py

```diff
@@ -113,3 +116,24 @@ def get(self, request, *args, **kwargs):
         'MODEL': model,
         '__doc__': ModelSchemaView.__doc__.format(model.__name__, model.__doc__)
     }))
+
+
+class ModelTypesView(views.APIView):
+
+    def __init__(self, *args, **kwargs):
+        super(ModelTypesView, self).__init__(*args, **kwargs)
+        yaml_file = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))) + '/share/models/creative.yaml'
```

Comment:
Use `os.path.join` over the `+` here. Rather than using multiple `os.path.dirname`s throw `../` into the path.